projects/ion/src/lib/input/input.component.ts

```diff
@@ -35,6 +37,7 @@ export class InputComponent {
   @Input() inputButton? = false;
   @Input() inputIconButton? = false;
   @Input() value = '';
+  @Input() inputType: InputType;
```

Comment:
```suggestion
  @Input() inputType: InputType = 'text';
```
I think this should be 'text' by default.